eng/targets/ReferenceAssembly.targets

```diff
@@ -47,6 +47,7 @@
     <PropertyGroup>
       <_RefSourceOutputPath>$([System.IO.Directory]::GetParent('$(MSBuildProjectDirectory)'))/ref/</_RefSourceOutputPath>
       <_RefSourceFileName>$(AssemblyName).$(TargetFramework).cs</_RefSourceFileName>
+      <_ManualRefSourceFileName>$(AssemblyName).Manual.cs</_ManualRefSourceFileName>
```

Comment:
Are there any typeforwards that are TFM specific? Like, in ns2.0 the type was still in assembly A, and then we moved it to assembly B for nca3.0?

Reply:
Not right now. I want to keep this simple for now, but I do recognize that there could potentially be TFM dependency manual content. If that arises, we'll need to make the Manual.cs files TFM specific. There are other clean up items I'd like to make so I'll file a followup issue.